g11n-ws/vip-common/src/main/java/com/vmware/vip/common/i18n/status/APIResponseStatus.java

```diff
@@ -32,5 +32,7 @@ public class APIResponseStatus {
     public static final Response INTERNAL_NO_RESOURCE_ERROR = new Response(404,
             "Not Found");
     public static final Response UNKNOWN_ERROR = new Response(520, "Unknown Error");
-    public static final Response TRANSLATION_COLLECT_REQUEST_SUCCESS = new Response(200, "Source collection request have received");
+    public static final Response TRANSLATION_COLLECT_REQUEST_SUCCESS = new Response(200, "Source collection request have received(please use translation-product-component-api to confirm it)");
+    public static final Response TRANSLATION_COLLECT_SOURCE_PARAM_IS_FAlSE = new Response(400, "Check received the sources status, please use translation-product-component-api to confirm it.");
```

Comment:
If change the code to 400, that means 'false' is invalid value, only support value 'true', and from the bug's description this is required. 

So I think change the response code to '400' and message to 'Incorrect collectsource(only allows true)' is enough. So you need revert the change for message COLLECTSOURCE_NOT_VALIDE_L10N and other changes accordingly.

Besides you can set it as 'true' by default.

Reply:
I change the code to 200, The COLLECTSOURCE_NOT_VALIDE can't change